src/EvoSC.Manialinks/Templates/Scripts/Checkbox.ms

```diff
@@ -0,0 +1,46 @@
+﻿<!--
+Void SetCheckboxState(Text idName, Boolean value) {
+    declare checkboxFrame <=> Page.MainFrame.GetFirstChild(idName) as CMlFrame;
+    declare IsChecked for checkboxFrame = value;
+    
+    declare bg <=> checkboxFrame.Controls[4] as CMlQuad;
+    declare icon <=> checkboxFrame.Controls[6] as CMlLabel;
+    
+    IsChecked = value;
+    
+    if (value) {
+        AnimMgr.Add(bg, """<quad opacity='1' />""", 500, CAnimManager::EAnimManagerEasing::ExpOut);
+        icon.SetText("");
+    } else {
+        AnimMgr.Add(bg, """<quad opacity='0' />""", 500, CAnimManager::EAnimManagerEasing::ExpOut);
```

Comment:
I found 320ms to be the sweetspot for transitions like these, but that's just a opinion